hack/ci/e2e.sh

```diff
@@ -153,29 +148,33 @@ run_tests() {
     -o=jsonpath='{range .items[*]}{.metadata.name}{"\t"}{.spec.taints}{"\n"}{end}' \
     | grep -cv "node-role.kubernetes.io/master" )"
 
-  # setting this env prevents ginkg e2e from trying to run provider setup
+  # setting this env prevents ginkgo e2e from trying to run provider setup
   export KUBERNETES_CONFORMANCE_TEST="y"
-  # run the tests
   ./hack/ginkgo-e2e.sh \
     '--provider=skeleton' "--num-nodes=${NUM_NODES}" \
     "--ginkgo.focus=${FOCUS}" "--ginkgo.skip=${SKIP}" \
     "--report-dir=${ARTIFACTS}" '--disable-log-dump=true'
 }
 
-# setup kind, build kubernetes, create a cluster, run the e2es
-main() {
```

Comment:
pointless main, as this isn't re-entrant, and we don't support `curl | sh` for this script.